kvbc/cmf/categorized_kvbc_msgs.cmf

```diff
@@ -22,39 +22,88 @@ Msg SharedValueData 4{
     list string categories_ids
 }
 
-Msg SharedUpdatesData 5{
+Msg SharedKeyValueUpdatesData 5{
     map string SharedValueData kv
     bool calculate_hash
 }
 
 
 # Updates info
 
-Msg MerkleKeyFlag 6{
+Msg MerkleKeyFlag 10{
```

Comment:
Are the ID gaps used to allow new messages of different types? If yes, maybe we can use a much bigger gaps, something like 100 or 1000. What do you think?

Reply:
I didn't see it complain so I guess it's allowed. I'll enlarge the gap